### ISSUE TITLE
Add feature to convert magnet links to .torrent files

### ISSUE THREAD

**Issue by tangestani** [mipstian/catch#130]
I really like the organization feature, but I haven't been able to use it ever since magnets links have become the new standard. I think it would be worthwhile to add the ability to convert magnet links to .torrent files before saving it to the appropriate location.

**Comment by tangestani** [comment]
Your solution does sound better. Only problem is that this feature would then be restricted to Transmission users.

**Comment by mipstian** [comment]
True, but transmission seems to be extremely popular, and nothing's stopping us from adding support for other clients if the demand is there. I guess we'll see how big the intersection of "i don't use transmission" and "i really like the organization feature" is, it might be nil for all I know :)
